kokoro/gcp_ubuntu/cmake/build.sh

```diff
@@ -23,16 +23,15 @@ set -x
 export PS4='[$(date -u "+%T %Z")] '
 
 export CMAKE_BIN="$(which cmake)"
-export CC="$(which clang-6.0)"
-export CXX="$(which clang++-6.0)"
+export CC="$(which clang)"
+export CXX="$(which clang++)"
```

Comment:
This seems like something that should be in the docker image env

Reply:
...which means they should be dropped here :-D

Reply:
```suggestion
```

Reply:
Can't figure out how to make GitHub mark a conversation unresolved :angry:

Reply:
This interface is honestly so confusing at times...